src/ctia/entity/event/crud.clj

```diff
@@ -4,12 +4,9 @@
    [ctia.entity.event.schemas
     :refer [Event PartialEvent]]
    [ctia.stores.es.crud :as crud]
-   [clj-momo.lib.es
-    [document :as document]
-    [schemas :refer [ESConnState SliceProperties]]
-    [slice :refer [get-slice-props]]]))
+   [clj-momo.lib.es.schemas :refer [ESConnState]]))
 
-(defn attach-bulk-fields [index event]
+'(defn attach-bulk-fields [index event]
```

Comment:
?

Reply:
not done in purpose :-)
But it clearly reveals that I can complete this `ns` clean up and remove it :-)